### PR TITLE
Some cleanups to make flake8 clean

### PR DESCRIPTION
**Pull request by timj**
Excluded some astropy helper and sphinx configuration files.

### CODE REVIEW

**Comment by timj** [comment]
This implements #121

File: .travis.yml [hunk 1]
```diff
@@ -78,16 +78,11 @@ matrix:
         - os: linux
           env: PYTHON_VERSION=3.5 NUMPY_VERSION=1.14
 
-        # Do a PEP8 test with pycodestyle
+        # Do a PEP8 test with flake8
         - os: linux
-          env: MAIN_CMD='pycodestyle pyvo --count' SETUP_CMD=''
+          env: MAIN_CMD='flake8 pyvo' SETUP_CMD=''
```
Comment (bsipocz) [inline .travis.yml]: 💯 for changing this and being more strict, it makes much sense for this package.

File: tests/testLicense.py [hunk 2]
```diff
@@ -4,13 +4,16 @@
 """
 from __future__ import print_function, division
 
-import os, sys, shutil, re, imp, glob
-import unittest, pdb
+import os
+import re
+import unittest
+import pdb  # noqa: F401
```
Comment (bsipocz) [inline tests/testLicense.py]: beyond the scope of this PR, but why is this needed?

File: .travis.yml (hunk 1 above)
Comment (timj) [reply .travis.yml]: I don't know enough about Travis to see how to make this target more efficient (if you look at the job it installs loads of packages including texlive and takes far longer than it needs to).

File: tests/testLicense.py (hunk 2 above)
Comment (timj) [reply tests/testLicense.py]: pdb is not used by the code so I have to decide whether to leave it in and mask it, or remove the line on the basis that we don't need to have it there at all. In the interests of minimizing the change I masked it but I'm happy to remove the pdb line completely.

Comment (timj) [reply tests/testLicense.py]: (I think the point is that some people like to leave pdb imports hanging around to make life imperceptibly easier when adding a debug breakpoint later on)

File: setup.cfg [hunk 3]
```diff
@@ -19,19 +19,14 @@ norecursedirs = build docs/_build
 [ah_bootstrap]
 auto_use = True
 
+[flake8]
+max-line-length = 110
+max-doc-length = 79
+exclude = __init__.py, setup_package.py, conf.py, distribute_setup.py, setup.py, ah_bootstrap.py
```
Comment (bsipocz) [inline setup.cfg]:
```suggestion
exclude = __init__.py, setup_package.py, conf.py, distribute_setup.py, setup.py, ah_bootstrap.py, astropy_helpers, version.py
```

File: .travis.yml (hunk 1 above)
Comment (bsipocz) [reply .travis.yml]: yes, those can be cleaned up. also there are a few unused files scattered around. I'll try to open a follow-up PR in the evening or during the weekend.

File: tests/testLicense.py (hunk 2 above)
Comment (bsipocz) [reply tests/testLicense.py]: yeah, but that shouldn't really live in the code similarly to random print statements that sole purpose it to make debugging easier. But this is also a topic for further cleanup.

Comment (timj) [reply tests/testLicense.py]: I'll remove it.

File: .travis.yml (hunk 1 above)
Comment (bsipocz) [reply .travis.yml]: given that there are examples and scripts ourside of the `pyvo` directory I would suggest though to run flake8 for everything (and explicitly list all generated or bundled external files to be skipped)

**Comment by timj** [comment]
@cbanek Checks are all green!

File: .travis.yml (hunk 1 above)
Comment (cbanek) [reply .travis.yml]: Okay I totally agree everything should be flake8 and would be happy to take that in another PR

**Comment by cbanek** [comment]
Thanks @timj!

File: .travis.yml (hunk 1 above)
Comment (timj) [reply .travis.yml]: Actually, `flake8` works without the `pyvo` argument so I can remove that and this will still be fine.

Comment (bsipocz) [reply .travis.yml]: yes, that's what I meant.

**Pull request merged**